src/Sulu/Bundle/PreviewBundle/Command/CacheClearCommand.php

```diff
@@ -22,6 +22,8 @@
 
 class CacheClearCommand extends BaseCacheClearCommand
 {
+    protected static $defaultName = 'cache:clear';
```

Comment:
The names are now initialized lazy, that's why they have to be defined static (so that they are available at runtime).